include_core/omrutilbase.h

```diff
@@ -159,6 +159,36 @@ J9CAS8Helper(volatile uint64_t *addr, uint32_t compareLo, uint32_t compareHi, ui
 #endif /* !OMR_ENV_DATA64 && (AIXPPC || LINUXPPC) */
 
 
+#if defined(__riscv) && defined(LINUX)
```

Comment:
I suggest to remove `&& defined(LINUX)` because:
1. the code it not really Linux specific, it is RISC-V specific (as far as I undestand)
2. Both `RiscvCAS32Helper()` and `RiscvCAS64Helper()` are used in `AtomicSupport.hpp` where their usage is guarded by only `#if defined(__riscv)` and therefore will fail to compile in cases where `LINUX` is not `#define`d. Note, that `LINUX` is not defined automagically on all Linux systems. Actually, I found that because non-OpenJ9 project using OMR failed to compile on RISC-V because of this.

Reply:
`LINUX ` is defined in OpenJ9 and not specific to OMR.  So I agree to remove `defined(LINUX)`. in this case.